client/app/components/viz/VariantInspectCard.vue

```diff
@@ -1248,6 +1352,16 @@ export default {
           }
         },2000)
       }
+      
+      if (self.$refs.popupPedigreeGenotypeViz) {
+        self.$refs.popupPedigreeGenotypeViz.update();
+      } else {
+        setTimeout(function() {
```

Comment:
We hit this else condition when the $refs variable is not yet initialized. But waiting 2 seconds doesn't guarantee that the variable will be initialized. I recommend removing the code in the else block.

Reply:
Okay lets talk about these comments tomorrow. This particular block is the same as the existing block above it which already was there. Maybe we refactor that part of this more generally. I would agree with your idea of trying to see if we can get these charts to be more "naturally" reactive.